app/code/core/Mage/Core/Model/Cache.php

```diff
@@ -664,6 +665,12 @@ public function cleanType($typeCode)
         $tags = $this->getTagsByType($typeCode);
         $this->clean($tags);
 
+        $path = self::XML_PATH_TYPES . '/' . $typeCode . '/tags';
+        $tagsConfig = Mage::getConfig()->getNode($path);
+        if ($tagsConfig) {
+            $this->save(Mage::getSingleton('core/date')->gmtDate(), (string)$tagsConfig . '_updated_at');
```

Comment:
the ID for the cache record is something like "BLOCK_HTML_updated_at", are we should this ID is ok? it looks too generic and short.